jetson/odrive_bridge/src/__main__.py

```diff
@@ -66,6 +72,16 @@ def lcmThreaderMan():
     lcm_1.subscribe("/drive_vel_cmd", drive_vel_cmd_callback)
     while True:
         lcm_1.handle()
+        global start_time
+        start_time = t.clock()
+        try:
+            publish_encoder_msg()
+        except NameError:

```

Comment:
when does this error happen?

Reply:
i dont really remember to be honest, i think this also gets thrown sometimes instead of attribute error depending on what it tries to do before switching to the disconnected state